eva-server/src/test/java/uk/ac/ebi/eva/server/ws/VariantWSServerTest.java

```diff
@@ -105,12 +105,12 @@ private void testGetVariantByIdRegionDoesntExistHelper(String testString) throws
 
     @Test
     public void testCheckVariantExistsDoesExist() throws URISyntaxException {
-        assertTrue(testCheckVariantExistsHelper("20:71822:C:G"));
+        testCheckVariantExistsHelper("20:71822:C:G");
```

Comment:
I removed them because I thought they were failing for a different reason than they are, I thought they were failing because there wasn't meant to be an assert there, and that all the asserts happen in the helper.